.github/workflows/test.yml

```diff
@@ -1,11 +1,11 @@
-name: "Tests"
+name: 'Tests'
 
 on: pull_request
 
 env:
-  OTP_VERSION: "24.0.1"
-  ELIXIR_VERSION: "1.12.1"
-  NODE_VERSION: "14"
+  OTP_VERSION: '24.0.1'
+  ELIXIR_VERSION: '1.12.1'
+  NODE_VERSION: '14'
```

Comment:
Why the change though?

Reply:
Reverted, my VS code `Prettier` plugin did it on save 😄